### PR TITLE
Added the sign up logic

### PR DESCRIPTION
**Pull request by Maadhav**

### CODE REVIEW

File: lib/ui/screens/login_screen.dart [hunk 1]
```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
 }
 
 class _LoginScreenState extends State<LoginScreen> {
-  FocusNode _passwordFocusNode;
-
+  FocusNode _passwordFocusNode, _emailFocusNode;
```
Comment (rafalbednarczuk) [inline lib/ui/screens/login_screen.dart]: Please keep these variable declarations separate

File: lib/ui/screens/login_screen.dart [hunk 2]
```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
...
+  bool isSignUp;
```
Comment (rafalbednarczuk) [inline lib/ui/screens/login_screen.dart]: Please use enum page {signUp, signIn} instead of bool

File: lib/ui/screens/login_screen.dart [hunk 3]
```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
...
     return Scaffold(
-        backgroundColor: Colors.transparent,
-        body: Stack(
-          children: <Widget>[
-            Positioned(
-              top: 0,
-              left: 0,
-              right: 0,
-              child: Image.asset("assets/pizza.jpg"),
+      backgroundColor: Colors.transparent,
```
Comment (rafalbednarczuk) [inline lib/ui/screens/login_screen.dart]: this line does nothing, please remove it

File: lib/ui/screens/login_screen.dart [hunk 4]
```diff
@@ -8,146 +8,188 @@ class LoginScreen extends StatefulWidget {
...
+                          onFieldSubmitted: (_) {
+                            FocusScope.of(context)
+                                .requestFocus(_emailFocusNode);
+                          },
                         ),
-                        TextFormField(
-                          focusNode: _passwordFocusNode,
+                      ),
+                      AnimatedContainer(
+                        duration: Duration(seconds: 1),
```
Comment (rafalbednarczuk) [inline lib/ui/screens/login_screen.dart]: please make animation faster: ~200 miliseconds

File: lib/ui/screens/login_screen.dart (hunk 1 above)
Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: Can I know the reason

File: lib/ui/screens/login_screen.dart (hunk 2 above)
Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: Ok

File: lib/ui/screens/login_screen.dart (hunk 4 above)
Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: Ok

File: lib/ui/screens/login_screen.dart (hunk 1 above)
Comment (rafalbednarczuk) [reply lib/ui/screens/login_screen.dart]: We use separate variable declarations style across whole project in different files. We can't use 2 different styles.

File: lib/ui/screens/login_screen.dart (hunk 2 above)
Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: what is the problem with using bool

Comment (rafalbednarczuk) [reply lib/ui/screens/login_screen.dart]: if isSignUp == true we know page is signUp, but if isSignUp == false it tells us nothing what page should be. enum tells there are only 2 pages and tells what are these pages

Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: isSignUp == false simply means login here

Comment (rafalbednarczuk) [reply lib/ui/screens/login_screen.dart]: We know the logic we want to implement so it's obvious for us. But if someone else takes a look into this code it is not so obvious. Also it's more convenient for future as we might add new pages like "I forgot password etc"

Comment (Maadhav) [reply lib/ui/screens/login_screen.dart]: I dont know how to use enum in dart

Comment (rafalbednarczuk) [reply lib/ui/screens/login_screen.dart]: Okay, I'll take care of this enum myself

**Review by Maadhav — COMMENTED** [review]
Now everything is perfect in the Login/Sign Up page according to me

**Review by rafalbednarczuk — APPROVED** [review]

**Pull request merged**